mmap-process/src/main/java/org/tools4j/process/IdleStrategy.java

```diff
@@ -0,0 +1,11 @@
+package org.tools4j.process;
```

Comment:
can you please name this ``org.tools4j.mmap.process``

Reply:
Plus everything else also prefix ``org.tools4j.mmap``

Reply:
Unless you plan to move process into its own repo --- then I am fine to leave as is for now

Reply:
moved to tools4j-process